### PR TITLE
Add scope and supporting script for RDP access to Windows tasks

### PR DESCRIPTION
**Pull request by SimonSapin**
Depended on by https://github.com/servo/servo/issues/26082
Depends on https://github.com/mozilla/community-tc-config/pull/252

### CODE REVIEW

File: config/roles.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@
     - project:servo:docker-worker-kvm:capability:privileged
     - docker-worker:cache:*
     - generic-worker:cache:*
+    - generic-worker:allow-rdp:proj-servo/*
```
Comment (SimonSapin) [inline config/roles.yml:28]: I’ve already applied this.

File: commands/rdp.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        task_id = task_id_or_url[len(prefix):]
+    else:
+        task_id = task_id_or_url
+
+    options = taskcluster.optionsFromEnvironment()
+    queue = taskcluster.Queue(options)
+    rdp_info = queue.getLatestArtifact(task_id, "project/servo/rdp-info")
+
+    supported_clients = [
+        ["xfreerdp", "/v:{host}:{port}", "/u:{username}", "/p:{password}"],
```
Comment (SimonSapin) [inline commands/rdp.py:18]: @paulrouget What RDP client do you use on mac? If it has CLI interface we could add it here.

Comment (paulrouget) [reply commands/rdp.py:18]: You could also create a .rdp file: https://docs.microsoft.com/en-us/windows-server/remote/remote-desktop-services/clients/rdp-files

```
auto connect:i:1
full address:s:{host}:{port}
username:s:Administrator
```

Not sure how the password can be stored.

Comment (SimonSapin) [reply commands/rdp.py:18]: Do you feel adding a script for this is worth it, if you still need to then manually open that file and copy-paste a password? Either way this seems like a different script, so let’s land this and we can always add something else later.

**Pull request merged**